src/reporter.ts

```diff
@@ -91,26 +91,35 @@ export default class SauceReporter extends SummaryFormatter {
 
   logTestCase(testCaseFinished: { testCaseStartedId: string }) {
     const testCaseAttempt = this.eventDataCollector.getTestCaseAttempt(testCaseFinished.testCaseStartedId);
-    const parsed = formatterHelpers.parseTestCaseAttempt({
-      snippetBuilder: this.snippetBuilder,
-      supportCodeLibrary: this.supportCodeLibrary,
-      testCaseAttempt
-    });
+    let parsed: any;
+    try {
+      parsed = formatterHelpers.parseTestCaseAttempt({
+        snippetBuilder: this.snippetBuilder,
+        supportCodeLibrary: this.supportCodeLibrary,
+        testCaseAttempt
+      });
+    } catch (e) {
+      console.error('failed to parse test data: ', e);
+      if (e instanceof TypeError) {
+        console.warn('\n\x1b[33m`paths` field might be set multiple times in your config. Please check your Cucumber.js config file.\n\x1b[0m');
```

Comment:
I don't think the log viewer in the sauce ui renders ansi colour codes, so this will be a little less readable on sauce. FYI, we explicitly disable ansi color codes in for playwright in the runner: https://github.com/saucelabs/sauce-playwright-runner/blob/c4a78ecb123c232c02bf1c2c84e6ea7418f0164b/src/playwright-runner.js#L430

Reply:
not sure what else I can do, just remove the colourized output and a newline

Reply:
Well placed whitespace is very effective for readability, imo. Don't think you lose that much by removing color.